Makefile

```diff
@@ -0,0 +1,33 @@
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+current_dir := $(dir $(mkfile_path))
+
+TYPO3_WEB_DIR := $(current_dir).Build/web
+TYPO3_PATH_ROOT := $(current_dir).Build/web
+# Allow different versions on travis
+TYPO3_VERSION ?= ~8.7
+typo3DatabaseName ?= "feuser_location_test"
+typo3DatabaseUsername ?= "dev"
+typo3DatabasePassword ?= "dev"
+typo3DatabaseHost ?= "127.0.0.1"
+
+.PHONY: install
+install: clean
+	COMPOSER_PROCESS_TIMEOUT=1000 composer require -vv --dev --prefer-dist --ignore-platform-reqs typo3/cms="$(TYPO3_VERSION)"
+	git checkout composer.json
+
+functionalTests:
+	typo3DatabaseName=$(typo3DatabaseName) \
```

Comment:
Is the indent here correct?

Tabs are used here.

Reply:
You have to use Tabs in Makefiles.

> We use space-only indentation in nearly all files. This includes all *.h, *.c, *.y files, except for the regex module. Makefile and ChangeLog files are excluded, since TAB characters are part of their format.


https://www.gnu.org/software/gnulib/manual/html_node/Indent-with-spaces-not-TABs.html